docs/dev/low-code/execute/schedules-page.md

```diff
@@ -125,7 +125,9 @@ To create a new schedule:
 
 <img src={useBaseUrl('/img/dev/low-code/schedule-create-job.png')} alt="Create Schedule window - Job tab" width="300"/>
 
-4. On the **Schedule** tab, enter a start and end time for the test. Below that, enter a length of time, in months, days, or hours.
+4. On the **Schedule** tab, enter a start and end time for the test.
+
```

Comment:
Suggestion to remove the extra space in between steps to maintain one line spacing
```suggestion
```